test/controllers/TaxCodeChangeControllerSpec.scala

```diff
@@ -18,10 +18,11 @@ package controllers
 
 import builders.{AuthBuilder, RequestBuilder}
 import mocks.MockTemplateRenderer
+import org.jsoup.Jsoup
 import org.mockito.Matchers.{any, eq => mockEq}
 import org.mockito.Mockito.when
 import org.scalatest.mock.MockitoSugar
-import org.scalatestplus.play.PlaySpec
+import org.scalatestplus.play.{OneAppPerSuite, PlaySpec}
```

Comment:
We have some imports that don't seem to be used anymore: eg `OneAppPerSuite` and `import org.mockito.Matchers.{eq => mockEq}
`